platform/linux-generic/odp_packet_io.c

```diff
@@ -2204,7 +2203,19 @@ int odp_pktin_queue_config(odp_pktio_t pktio,
 		return -1;
 	}
 
-	num_queues = param->classifier_enable ? 1 : param->num_queues;
+	if (param->classifier_enable) {
+		num_queues = 1;
+
+		if (param->num_queues > 1) {
```

Comment:
Since API says that: "When classifier is enabled in odp_pktin_queue_config() this value is ignored, ..." value of param->num_queues could be anything - also zero. So, may be copy is needed when (param->num_queues != 1).

Reply:
Yes, good point. Fixed in v3.